init_db.py

```
@@ -0,0 +1,23 @@
+    # Give the database a moment to start up
+    time.sleep(5)
+    
+    try:
+        # The engine is already created in engine.py
+        # This will attempt to connect to the database
+        with engine.connect() as connection:
+            print("Database connection successful.")
+            print("Creating tables...")
+            SQLModel.metadata.create_all(engine)
+            print("Tables created successfully.")
+    except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)